core/src/test/scala/unit/kafka/log/LogCleanerManagerTest.scala

```diff
@@ -501,7 +501,7 @@ class LogCleanerManagerTest extends Logging {
     val pausedPartitions = cleanerManager.pauseCleaningForNonCompactedPartitions()
     // Log truncation happens due to unclean leader election
     cleanerManager.abortAndPauseCleaning(log.topicPartition)
-    cleanerManager.resumeCleaning(Set(log.topicPartition).asJava)
+    cleanerManager.resumeCleaning(util.Set.of(log.topicPartition))
     // log cleanup finishes and pausedPartitions are resumed
     cleanerManager.resumeCleaning(pausedPartitions.asScala.map(_.getKey).toSet.asJava)
```

Comment:
Could we eliminate the conversion?
Additionally, there are some similar patterns in this test; please clean them up.